hbase-shaded/hbase-shaded-testing-util/src/main/resources/org/apache/hadoop/hbase/shaded/org/mortbay/jetty/webapp/webdefault.xml

```diff
@@ -0,0 +1,430 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+Copyright (C) Jetty Authors
+
+Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this is the wrong ALv2 header I think? it should be the one that says it's licensed under those terms, but not the "licensed toe the ASF under a contributor agreement" bit.

like over here:
https://github.com/apache/hbase/blob/909140980045c4bb75908be36142841dafbf67ab/hbase-common/src/main/java/org/apache/hadoop/hbase/io/LimitInputStream.java#L4

Reply:
I didn't notice they are different. Thanks.